packages/ui/src/components/accordion/accordion.vue

```diff
@@ -90,29 +88,25 @@
 
   const classes = computed(() => {
     return {
-      'acv-accordion': true,
       [`acv-accordion--size-${size}`]: size,
     };
   });
 </script>
 
 <template>
   <div
-    class="acv accordion"
+    class="acv-accordion"
     :class="classes"
   >
     <slot />
   </div>
 </template>
 
 <style scoped>
-.acv {
-  &.accordion {
+  .acv-accordion {
     font-weight: var(--acv-font-weight-strong);
-    color: var(--acv-accordion-color);
+    color: var(--acv-accordion-color, var(--acv-color-base-primary));
     border: 2px solid;
     border-radius: 0.5rem;
-    overflow: hidden;
```

Comment:
need to be more strict with this rule. We must try our best to not cut users markups. It's absolutely OK if user has some tooltip that overflows over accordion edges